### PR TITLE
🦊 Add gitlab support

### PR DESCRIPTION
**Pull request by mxbaylee**

### CODE REVIEW

File: settings/gitlab.mdx [hunk 1]
```diff
@@ -8,30 +8,78 @@ icon: 'gitlab'
...
+  </Step>
+
+  <Step title="Create the webhook">
+    a. Navigate to your GitLab's project `Settings` > `Webhooks`.
+
+    b. In the "URL" field, enter the endpoint `https://leaves.mintlify.com/gitlab-webhook`
+
+    c. Name the webhook "Mintlify".
 
+    d. Paste the Webhook token generated after completing the previous steps.
```
Comment (hahnbeelee) [inline settings/gitlab.mdx]: I think we should add a screenshot of the product here highlighting the "Show Webtoken".

Comment (mxbaylee) [reply settings/gitlab.mdx]: abs, on it!

Comment (mxbaylee) [reply settings/gitlab.mdx]: updated!

**Comment by ricardonunez-io** [comment]
Thank you

**Review by dks333 — APPROVED** [review]
lgtm

**Pull request merged**